recipes/protobuf/all/conanfile.py

```diff
@@ -128,6 +128,10 @@ def _configure_cmake(self):
             cmake.definitions["protobuf_MSVC_STATIC_RUNTIME"] = "MT" in runtime
         if Version(self.version) < "3.18.0" and self._is_clang_cl:
             cmake.definitions["CMAKE_RC_COMPILER"] = os.environ.get("RC", "llvm-rc")
+        if is_apple_os(self) and self.settings.os in ["iOS", "tvOS"] and self.conf.get("tools.apple:enable_bitcode", check_type=bool):
+            cmake.definitions["CONAN_C_FLAGS"] = "-fembed-bitcode"
+            cmake.definitions["CONAN_CXX_FLAGS"] = "-fembed-bitcode"
+            cmake.definitions["CONAN_LDFLAGS"] = "-fembed-bitcode"
```

Comment:
It's not the conan v2 way to add bitcode, migrate this recipe to conan v2.

Reply:
Hi! We're porting this recipe to v2 by ourselves, so there would be no need to do it by yourself @kletoz, we'll let you know once this has been merged so that you can rebase your fix on top of the new code